tests/first-facets-distribution.tests.ts

```diff
@@ -0,0 +1,148 @@
+import { dataset, meilisearchClient, HOST, API_KEY } from './assets/utils'
+import { instantMeiliSearch } from '../src'
+
+describe('Firt facet distribution', () => {
+  beforeAll(async () => {
+    const deleteTask = await meilisearchClient.deleteIndex('movies')
+    await meilisearchClient.waitForTask(deleteTask.taskUid)
+    await meilisearchClient
+      .index('movies')
+      .updateFilterableAttributes(['genres'])
+    const documentsTask = await meilisearchClient
+      .index('movies')
+      .addDocuments(dataset)
+    await meilisearchClient.index('movies').waitForTask(documentsTask.taskUid)
+  })
+
+  test('creation of facet distribution without facets', async () => {
+    const searchClient = instantMeiliSearch(HOST, API_KEY)
+    const response = await searchClient.search([
+      {
+        indexName: 'movies',
+        params: {
+          facets: [],
+          query: '',
+        },
+      },
+    ])
+    expect(response.results[0].facets).toEqual({})
+  })
+
+  test('creation of facet distribution without facets and with keepZeroFacets to true', async () => {
+    const searchClient = instantMeiliSearch(HOST, API_KEY, {
+      keepZeroFacets: false,
+    })
```

Comment:
`with keepZeroFacets to true` in the title of the test, but `keepZeroFacets: false` in the code 👀